cache/test_cache_control.py

```diff
@@ -1291,110 +1291,8 @@ class StoringResponsesWithSetCookieHeaderDefaultCache(TestCacheControl, SingleTe
-    should_be_cached = False
-
-
-class StoringResponsesWithSetCookieHeaderNoTransformCache(TestCacheControl, SingleTest):
-    """
-    This test sends two requests to the same resource, which responds with no-transform
-    cache-control and set-cookie headers, and check that second request wasn't serviced
-    from the cache.
-    """
-
-    tempesta_config = """
-        cache_fulfill * *;
-        """
-    response_headers = {"Set-Cookie": "session=1", "Cache-control": "no-transform"}
-    second_request_headers = {"Cookie": response_headers["Set-Cookie"]}
-    should_be_cached = True
-
-
-class StoringResponsesWithSetCookieHeaderPrivateCache(TestCacheControl, SingleTest):
-    """
-    This test sends two requests to the same resource, which responds with private cache-control
-    and set-cookie headers, and check that second request wasn't serviced from the cache.
-    """
-
-    tempesta_config = """
-        cache_fulfill * *;
-        """
-    response_headers = {"Set-Cookie": "session=1", "Cache-control": "private"}
-    second_request_headers = {"Cookie": response_headers["Set-Cookie"]}
-    should_be_cached = False
-
-
-class StoringResponsesWithSetCookieHeaderProxyRevalidateCache(TestCacheControl, SingleTest):
-    """
-    This test sends two requests to the same resource, which responds with proxy-revalidate
-    cache-control and set-cookie headers, and check that second request was serviced
-    from the cache.
-    """
-
-    tempesta_config = """
-        cache_fulfill * *;
-        """
-    response_headers = {"Set-Cookie": "session=1", "Cache-control": "proxy-revalidate, max-age=1"}
-    second_request_headers = {"Cookie": response_headers["Set-Cookie"]}
-    should_be_cached = True
-    sleep_interval = None
-
-
-class StoringResponsesWithSetCookieHeaderMaxAgeCache(TestCacheControl, SingleTest):
-    """
-    This test sends two requests to the same resource, which responds with max-age cache-control
-    and set-cookie headers, and check that second request wasn't serviced from the cache.
-    """
-
-    tempesta_config = """
-        cache_fulfill * *;
-        """
-    response_headers = {"Set-Cookie": "session=1", "Cache-control": "max-age=5"}
-    second_request_headers = {"Cookie": response_headers["Set-Cookie"]}
-    should_be_cached = True
-
-
-class StoringResponsesWithSetCookieHeaderSMaxAgeCache(TestCacheControl, SingleTest):
-    """
-    This test sends two requests to the same resource, which responds with s-maxage
-    cache-control and set-cookie headers, and check that second request was serviced
-    from the cache.
-    """
-
-    tempesta_config = """
-        cache_fulfill * *;
-        """
-    response_headers = {"Set-Cookie": "session=1", "Cache-control": "s-maxage=1"}
-    second_request_headers = {"Cookie": response_headers["Set-Cookie"]}
-    should_be_cached = True
-    sleep_interval = None
+    def test(self):
+        self.disable_deproxy_auto_parser()
```

Comment:
Please add header deletion `set-cookie` header in DeproxyAutoParser [here](https://github.com/tempesta-tech/tempesta-test/blob/262877a4ef72308f247e94b7f147e64db4220d52/framework/deproxy_auto_parser.py#L164) and enable it for this test